package.tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
   "compilerOptions": {
     "module": "esnext",
-    "target": "esnext",
-    "jsx": "preserve",
+    "target": "ES5",
```

Comment:
Hmm... I'm curious about this change. Will this compile to ESM?

Reply:
Yep, that's they they have module and target fields. Module defines what module system to compile down to (esnext === ESM and also commonjs).

Target is more like babel, to what language spec to transpile down to. So having Object.assign() instead of `{ ...foo }`.

So, in short, yes.

Reply:
🤦 Right, forgot about the separate module field. Thanks!